index.d.ts

```diff
@@ -2387,6 +2387,10 @@ export interface SelectMenuProps extends Omit<PopoverProps, 'position' | 'conten
    * When true, show the filter.
    */
   hasFilter?: boolean
+  /**
+   * @default true. When true, auto focuses on the search/filter bar. 
```

Comment:
Can we move the description on a separate line from the `@default` JSDoc attribute? I'm thinking that shows up weird in your IDE:

<img width="482" alt="image" src="https://user-images.githubusercontent.com/11774799/194661171-8649e027-55b7-4c2d-aa97-09b4d14f093a.png">

vs the expected:

<img width="483" alt="image" src="https://user-images.githubusercontent.com/11774799/194661491-6d9f7a3b-78ed-471c-bc41-01ff2782b34c.png">

Reply:
i see, thanks for the note!